docs/autoloop.md

```diff
@@ -0,0 +1,114 @@
+```
+loop setparams --autobudget={budget in satoshis}
+```
+
+Your autoloop budget can optionally be paired with a start time, which determines the time from which we will count autoloop swaps as being part of the budget. If this value is zero, it will consider all automatically dispatched swaps as being part of the budget. 
+
+The start time is expressed as a unix timestamp, and can be set using the `setparams` loop command:
+```
+loop setparams --budgetstart={start time in seconds}
+```
+
+If your autolooper has used up its budget, and you would like to top it up, you can do so by either increasing the overall budget amount, or by increasing the start time to the present. For example, if you want to set your autolooper to have a budget of 100k sats for the month, you could set the following:
+```
+loop setparams --autobudget=100000 --autostart={beginning of month ts}
+```
+
+## Dispatch Control
+Configuration options are also exposed to allow you to control the rate at which swaps are automatically dispatched, and the autolooper's propensity to retry channels that have previously failed. 
+
+### In Flight Limit
+The number of swaps that the autolooper will dispatch at a time is controlled by the `autoinflight` parameter. The default value for this parameter is 1, and can be increased if you would like to perform more automated swaps simultaneously. If you have set a very high sweep target for your automatically dispatched swaps, you may want to increase this value, because the autolooper will wait for the swap to fully complete, including the sweep confirming, before it dispatches another swap. 
+
+```
+loop setparams --autoinflight=2
+```
+
+### Failure Backoff
+Sometimes loop out swaps fail because they cannot find an off-chain route to the server. This may happen because there is a temporary lack of liquidity along the route, or because the peer that you need to perform a swap with simply does not have a route to the loop server's node. These swap attempts cost you nothing, but we set a backoff period so that the autolooper will not continuously attempt to perform swaps through a very unbalanced channel that cannot facilitate a swap. 
+
+The default value for this parameter is 24hours, and it can be updated as follows:
+```
+loop setparams --failurebackoff={backoff in seconds}
+```
+
+## Manual Swap Interaction
+The autolooper will not dispatch swaps over channels that are already included in manually dispatched swaps - for loop out, this would mean the channel is specified in the outgoing channel swap, and for loop in the channel's peer is specified as the last hop for an ongoing swap. This check is put in place to prevent the autolooper from interfering with swaps you have created yourself. If there is an ongoing swap that does not have a restriction placed on it (no outgoing channel set, or last hop), then the autolooper will take no action until it has resolved, because it does not know how that swap will affect liquidity balances. 
```

Comment:
General nit (non-blocking): For further review and merging of PRs, it's generally useful to also try to adhere to the 80 character line length limit where possible (links and tables often don't fit but flow text looks nicer IMO).

Reply:
makes sense, wrapped!